converter.py

```diff
@@ -73,7 +74,14 @@ def get_dump_name(filename, extension):
             for i, p in enumerate(data)
         },
         ".json",
-    )
+    ),
+    (
+        'local fixture config',
+        paths.experiment_local_fixturename,
+        FixtureFactory.serializer.load_first,
+        None,
+        ".json",
```

Comment:
I think it's better to write it back into the file
```suggestion
        None,
```
The reason @skymandr needed name change was that it was called `.npy` before which most def not a json, but `.fixture` could be anything.